extensions/ANGLE/ANGLE_depth_texture.txt

```diff
@@ -17,20 +19,20 @@ Contributors
 
 Contact
 
-    Daniel Koch, TransGaming (daniel 'at' transgaming.com)
+    Shannon Woods, TransGaming (shannon 'dot' woods 'at' transgaming.com)
 
 Status
 
     Implemented in ANGLE.
 
 Version
 
-    Last Modifed Date: June 4, 2012
-    Revision: #3
+    Last Modified Date: February 25, 2013
+    Revision: #4
 
 Number
 
-    OpenGL ES Extension #138
```

Comment:
Was this intentional? I think the extension number should stay.

Reply:
Oops, sorry, not intentional. Replaced, and also updated contact info and email address.